x-pack/test_serverless/functional/test_suites/common/reporting/management.ts

```diff
@@ -51,40 +54,35 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     };
 
     // Kibana CI and MKI use different users
-    const TEST_USERNAME = config.get('servers.kibana.username');
-    const TEST_PASSWORD = config.get('servers.kibana.password');
-
     before('initialize saved object archive', async () => {
+      roleName = 'admin';
+      roleAuthc = await svlUserManager.createApiKeyForRole(roleName);
       // add test saved search object
       await kibanaServer.importExport.load(savedObjectsArchive);
     });
 
     after('clean up archives', async () => {
       await kibanaServer.importExport.unload(savedObjectsArchive);
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
     });
 
     // Cant auth into the route as it's structured currently
     xit(`user sees a job they've created`, async () => {
       const {
         job: { id: jobId },
-      } = await reportingAPI.createReportJobInternal(CSV_REPORT_TYPE_V2, job);
+      } = await reportingAPI.createReportJobInternal(CSV_REPORT_TYPE_V2, job, roleAuthc);
 
       await navigateToReportingManagement();
       await testSubjects.existOrFail(`viewReportingLink-${jobId}`);
     });
 
     // Skipping test for now because functionality is not yet possible to test
     xit(`user doesn't see a job another user has created`, async () => {
```

Comment:
I'm not sure why we have skipped tests, but there isn't an issue correlated with these. Filed: https://github.com/elastic/kibana/issues/186558

Reply:
https://github.com/elastic/kibana/pull/184819/commits/033c7fa4f3c33d0e389b84ea3852f565177bd9b3